02.calendar/calender.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("y:", "m:")
+y = params["y"]
+m = params["m"]
+
+target_date = if y.nil? && m.nil?
+              Date.today
+            elsif y.nil?
+              Date.new(Date.today.year, m.to_i)
+            else
+              Date.new(y.to_i, m.to_i)
+            end
+
+puts target_date.strftime("%-m月 %Y").center(21)
+puts " 日 月 火 水 木 金 土"
+last_date = Date.new(target_date.year, target_date.month, -1).day
+first_date =  Date.new(target_date.year, target_date.month,)
```

Comment:
最後が`,`で終わってるのが不自然です。そもそも

```suggestion
first_date =  Date.new(target_date.year, target_date.month, 1)
```

のように日にちも省略せずに書いた方がわかりやすいコードになると思います。